conformance_monitoring_operations/custom_signals.py

```diff
@@ -35,9 +32,12 @@ def process_telemetry_conformance_message(sender, **kwargs):
             flight_declaration_id=flight_declaration_id
```

Comment:
Interesting stuff here :-D

I guess it is some python interpolation? Just looks a bit weird having flight_declaration_id=flight_declaration_id

Reply:
yeah this is just to format the string :)